ProjectSettings/ProjectSettings.asset

```diff
@@ -379,7 +379,6 @@ PlayerSettings:
   XboxOnePersistentLocalStorageSize: 0
```

Comment:
I want not to add this file to this commit unless this file is related to this change.